src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -295,6 +295,7 @@ public void reloadPage() {
     public <T extends AppPage> T goToPreviousPage(Class<T> typeOfPreviousPage) {
         browser.driver.navigate().back();
         waitForPageToLoad();
+        switchToNewWindow();
```

Comment:
Is this change required? It seems like the goToPreviousPage function might be called elsewhere, and that this statement will change the behavior.